scribus/text/screenpainter.cpp

```diff
@@ -179,15 +186,15 @@ void ScreenPainter::drawGlyph(const GlyphCluster& gc)
 			outline.addQuadPoint(1, -10, 1, -10, 0, -10, 0, -10);
 		}
 		chma.scale(gc.scaleH() * fontSize() / 10.0, gc.scaleV() * fontSize() / 10.0);
-		outline.map(chma * chma4);
+        outline.map(chma*chma4);
 		m_painter->setupPolygon(&outline, true);
 		QColor oldBrush = m_painter->brush();
 		m_painter->setBrush(gc.hasFlag(ScLayout_SuppressSpace) ? Qt::green
 															   : PrefsManager::instance()->appPrefs.displayPrefs.controlCharColor);
 		m_painter->setBrush(PrefsManager::instance()->appPrefs.displayPrefs.controlCharColor);
 		if (stroke)
 		{
-			QColor tmp = m_painter->pen();
+            QColor tmp = m_painter->pen();
```

Comment:
same above, as general we use 4 tabs for indentation, please adjust the code style as screenshot below.
![editor](https://cloud.githubusercontent.com/assets/1632566/17165915/d3f85518-53e4-11e6-8053-eaa3816cc383.jpg)